### PR TITLE
Adding UmiJS as a single-spa user.

### PR DESCRIPTION
**Pull request by internettrans**
Alipay is the primary company behind https://github.com/umijs/qiankun.

@kuitos @deturium can you please confirm if it is okay for us to add Alipay as a user of single-spa? This would would make Alipay show on two pages:

- Home page (near bottom) https://zh-hans.single-spa.js.org/
- List of single-spa users https://zh-hans.single-spa.js.org/users/

### CODE REVIEW

File: website/src/data/users.js
Comment (internettrans) [inline website/src/data/users.js:24]: Not everything can be pinned anymore

**Comment by kuitos** [comment]
hi @joeldenning 

It is our great honor to be a member of the single-spa user. But we need to file the legal declaration with the company for the external disclosure of the trademark. Currently, we have submitted the corresponding process and are in the process of approval. 

I will contact you as soon as possible after the approval is completed.

**Comment by internettrans** [comment]
👍 sounds good

**Review by TheMcMurder — APPROVED** [review]
Let's merge as soon as Alipay has the legal stuff in place for us to merge.

**Comment by internettrans** [comment]
And if things don't work out and you do not get approval, that is completely okay!

**Comment by kuitos** [comment]
Sorry for my late replay.
I have to tell you the bad news, the company legal requirements to share trademarks must be commercial companies, who could provide a company license to obtain approval, and the use of trademarks is a term, such as a maximum of one year.

Maybe we could change the user from alipay to umi? And the logo of umi org can be used at will.

**Comment by internettrans** [comment]
@kuitos I have updated this to reference umijs. Could you please approve?

**Review by kuitos — APPROVED** [review]

**Comment by kuitos** [comment]
@joeldenning approved~

**Pull request merged**
